Automattic/newspack-popups
# Add popup preview in the editor

### All Submissions:






### Changes proposed in this Pull Request:

This adds a preview button which enables previewing a popup in the editor:

![Screen Capture on 2020-01-27 at 12-29-59](https://user-images.githubusercontent.com/7383192/73171435-04d40d00-4101-11ea-93c0-d7f6f7b20318.gif)


Closes #2 

### How to test the changes in this Pull Request:

1. open popup editor
2. click "Preview" in sidebar's Pop-up Settings
3. a preview should be displayed

### Other information:
